test/unit/org/apache/cassandra/db/marshal/TypeValidationTest.java

```diff
@@ -19,16 +19,19 @@
 package org.apache.cassandra.db.marshal;
 
 import org.apache.cassandra.Util;
+import org.apache.cassandra.io.util.DataOutputPlus;
```

Comment:
Not related to the changes, but we could fix the license header to be a block comment instead of JavaDoc, replacing `/**` by `/*` in the very first line of the file.

Reply:
Good idea, done.